docs/releases/2.7.3.rst

```diff
@@ -0,0 +1,114 @@
+=====================================
+Welcome to the new Pootle 2.7.3 final
+=====================================
+
+*Released on 20 November 2015*
+
+Bugfix release for 2.7.2.
+
+
+Changes in Requirements
+=======================
```

Comment:
Ok, will remove the section then.

I haven't included the **allauth** requirement because that is already automatically handled by `pip`.